<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>785</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>323232</width>
    <height>323232</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>STIX data parser and viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/viewer.qrc">
    <normaloff>:/Icons/images/app.svg</normaloff>:/Icons/images/app.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Packets</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QTreeWidget" name="packetTreeWidget">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1000000</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <column>
             <property name="text">
              <string notr="true">Timestamp</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Description</string>
             </property>
            </column>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTreeWidget" name="paramTreeWidget">
            <property name="autoExpandDelay">
             <number>-1</number>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <attribute name="headerCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="headerDefaultSectionSize">
             <number>170</number>
            </attribute>
            <attribute name="headerHighlightSections">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Description</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Raw</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Eng. Value</string>
             </property>
            </column>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>plot parameters
</string>
       </property>
       <attribute name="title">
        <string>Plot</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QComboBox" name="xaxisComboBox">
            <item>
             <property name="text">
              <string>Parameter order as X</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Timestamps as X</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Histogram</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Data source:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>In</string>
            </property>
           </widget>
          </item>
          <item row="0" column="9">
           <widget class="QLineEdit" name="styleEdit">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="0" column="10">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>25</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="paramNameEdit">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QComboBox" name="comboBox">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="modelColumn">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>the same packet</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>all packets</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QLabel" name="descLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>0</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="8">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Curve Style:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="11">
           <widget class="QPushButton" name="plotButton">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Plot</string>
            </property>
           </widget>
          </item>
          <item row="0" column="12">
           <widget class="QPushButton" name="savePlotButton">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QComboBox" name="dataTypeComboBox">
            <item>
             <property name="text">
              <string>Raw values</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Engineering values</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="13">
           <widget class="QPushButton" name="exportButton">
            <property name="text">
             <string>Export data</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1148</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
    <addaction name="actionLog"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionSet_IDB"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Plot"/>
    <addaction name="actionPaste"/>
    <addaction name="actionLoad_mongodb"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuAction"/>
   <addaction name="menuSetting"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="action_Plot"/>
   <addaction name="actionPaste"/>
   <addaction name="actionLoad_mongodb"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionSet_IDB">
   <property name="text">
    <string>Set &amp;IDB</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Sa&amp;ve</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Show Log</string>
   </property>
  </action>
  <action name="action_Plot">
   <property name="icon">
    <iconset resource="resources/viewer.qrc">
     <normaloff>:/Icons/images/graph.svg</normaloff>:/Icons/images/graph.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Plot</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources/viewer.qrc">
     <normaloff>:/Icons/images/paste.svg</normaloff>:/Icons/images/paste.svg</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Read raw data from the clipboard</string>
   </property>
  </action>
  <action name="actionLoad_mongodb">
   <property name="icon">
    <iconset resource="resources/viewer.qrc">
     <normaloff>:/Icons/images/mongodb.svg</normaloff>:/Icons/images/mongodb.svg</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/viewer.qrc"/>
 </resources>
 <connections/>
</ui>
